src/iceth-client/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "iceth-client"
+version = "0.1.0"
```

Comment:
version and edition should be inherited from workspace

Reply:
Done